themes/small-apps/assets/scss/pages/_blog.scss

```diff
@@ -95,9 +95,12 @@
 		.post-body {
 			.feature-image {
 				margin-bottom: 30px;
+                margin-left: 40%;
+                margin-right: auto;
+                width: 40%;
 
 				img {
-					width: 100%;
+					height: 75px;
```

Comment:
Do we only want the pictures to be 75px high?

Reply:
It was an attempt to have the feature images be somewhat reasonable. This didn't work out so I simply removed them, I think it looks a lot better and is much more readable.